tests/unit/test_algebraic_expression.cpp

```diff
@@ -326,7 +326,6 @@ TEST_F(AlgebraicExpressionTest, Exp_OP_ADD) {
 	GrB_Matrix_free(&B);
 	GrB_Matrix_free(&C);
 	GrB_Matrix_free(&res);
-	AlgebraicExpressionNode_Free(exp);
```

Comment:
These are leaks, which I don't think are worth caring about in unit tests. I'll revert these changes and bring back `AlgebraicExpressionNode_Free` (which is only used in this file) if you prefer, though!

Reply:
That's all right, no harm leaking in unit tests